daemon/src/routes_maker.rs

```diff
@@ -4,11 +4,15 @@ use crate::model::{Usd, WalletInfo};
 use crate::to_sse_event::ToSseEvent;
 use anyhow::Result;
 
```

Comment:
Can we remove this newline? rustfmt doesn't format import blocks across empty lines.